lib/app.js

```diff
@@ -24,17 +24,28 @@ app.use('/api', ensureAuth);
 // and now every request that has a token in the Authorization header will have a `req.userId` property for us to see who's talking
 app.get('/api/test', (req, res) => {
   res.json({
-    message: `in this proctected route, we get the user's id like so: ${req.userId}`
+    message: `in this proctected route, we get the user's id like so: ${req.userId}`,
   });
 });
 
-app.get('/animals', async(req, res) => {
+app.get('/kitkats', async (req, res) => {
   try {
-    const data = await client.query('SELECT * from animals');
-    
+    const data = await client.query('SELECT * from kitkats');
+
     res.json(data.rows);
-  } catch(e) {
-    
+  } catch (e) {
+    res.status(500).json({ error: e.message });
+  }
+});
+
+app.get('/kitkats/:id', async (req, res) => {
+  try {
+    const id = req.params.id;
+
+    const data = await client.query(`SELECT * from kitkats where id=${id}`);
```

Comment:
It's best to sanitize the inputs. This can be done by changing id to `id=$1` and adding an array of the `[id]` after the SQL query :)